frontend/packages/kubevirt-plugin/src/k8s/wrapper/vm/combined-disk.ts

```diff
@@ -121,7 +127,7 @@ export class CombinedDisk {
       (dataVolumeWrapper) => dataVolumeWrapper.getStorageClassName(),
     );
 
-  getPVCName = (source?: StorageUISource) => {
+  getPVCNameBySource = (source?: StorageUISource) => {
     const resolvedSource = source || this.source;
```

Comment:
can we remove this line and just use the function parameter and make it non optional?

I guess it makes it less readable and does not have a use case at the moment

Reply:
done